### PR TITLE
Modelling of Nephio API objects

### PR DESCRIPTION
**Pull request by ciaranjohnston**
After some reviews in the modelling (WG4) meetings somewhat of a consensus has been reached on a view of the Nephio models, and this is the attempt to describe diagrammatically at a hight level what is currently there.

### CODE REVIEW

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: content/en/docs/apis/_index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+description: Reference for the Nephio models and APIs
+---
+
+# Nephio APIs
+
+## Overview
+
+Nephio APIs primarily consist of a collection of Go API objects, CRDs and other KRM types,
+specified and maintained in the [Nephio API repository](https://github.com/nephio-project/api).
+this page aims to give a high level overview of the objects that are available and their
```
Comment (efiacor) [inline content/en/docs/apis/_index.md:14]:
```suggestion
This page aims to give a high level overview of the objects that are available and their
```

File: content/en/docs/apis/_index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+relationships, and is based on an
+[original document](https://docs.google.com/document/d/1-5nlpY4FbuhWtdKTvIqPOv4bWmA6zx6TdHoEfk9Jc_Q/edit)
+developed ny Tal Liron.
```
Comment (efiacor) [inline content/en/docs/apis/_index.md:17]:
```suggestion
developed by Tal Liron.
```

File: content/en/docs/apis/_index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    click DataNetwork "https://doc.crds.dev/github.com/nephio-project/api/req.nephio.org/DataNetwork/v1alpha1@v2.0.0" "DataNetwork"
+    style NetworkInstance fill:#CCCCFF
+    click NetworkInstance "https://doc.crds.dev/github.com/nephio-project/api/req.nephio.org/NetworkInstance/v1alpha1@v2.0.0" "NetworkInstance"
+    style Pool fill:#CCCCFF
+    click Pool "https://github.com/nephio-project/api/blob/main/workload/v1alpha1/nf_deployment_types.go#L165" "Pool"
+
+```
+
+
+## Porch
```
[the remark this answers is not in the document]
Comment (ciaranjohnston) [reply content/en/docs/apis/_index.md:68]: I think having a recognition that content is needed here is good. I would work on this over the next few days and make a new PR for that

File: content/en/docs/apis/_index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+this page aims to give a high level overview of the objects that are available and their
+relationships, and is based on an
+[original document](https://docs.google.com/document/d/1-5nlpY4FbuhWtdKTvIqPOv4bWmA6zx6TdHoEfk9Jc_Q/edit)
+developed ny Tal Liron.
+
+The aim is to keep these diagrams as simple as possible for now and only convey the important aspects of the modelled entities. As such they are intended to give a high-level overview of the entities and relationships that can be accessed and modified via the Nephio API, and provide reference to detailed documentation, generated from the code, where available.
+
+
+## Topology and Network APIs
+
```
[the remark this answers is not in the document]
Comment (ciaranjohnston) [reply content/en/docs/apis/_index.md:23]: I've done that in the latest commit, thanks!

**Pull request merged**